apps/client/src/routes/dashboards/dashboards-index/components/getting-started.tsx

```diff
@@ -0,0 +1,160 @@
+import { memo } from 'react';
+import {
+  Box,
+  Button,
+  ColumnLayout,
+  ContentLayout,
+  ExpandableSection,
+  Link,
+  SpaceBetween,
+} from '@cloudscape-design/components';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { useApplication } from '~/hooks/application/use-application';
+import { CREATE_DASHBOARD_HREF } from '~/constants';
+import { gettingStartedColumnsTypes } from './types';
+import Dashboard from '../assets/Dashboard.svg';
+import Widget from '../assets/Widget.svg';
+import AssetsSearch from '../assets/AssetsSearch.svg';
+import './styles.css';
+
+const ValueWithLabel = ({
+  label,
+  children,
+}: {
+  label: string;
+  children: string;
+}) => (
+  <Box margin={{ bottom: 'xxl' }}>
+    <Box variant="awsui-key-label">{label}</Box>
+    <Box>{children}</Box>
+  </Box>
+);
+
+const GettingStarted = () => {
```

Comment:
typically memo is added here not while exporting, should not matter, just FYI for future reference

Reply:
Yes, previously I added it here only but due to your previous comment 'use arrow function'. If I used the arrow function then it throws a lint error. So I used default export at the bottom and used the arrow function here. :